packages/p2p/src/components/order-details/order-details.jsx

```diff
@@ -175,12 +174,6 @@ const OrderDetails = observer(() => {
                         onClickConfirm={() => order_store.confirmOrder(is_buy_order_for_user)}
                         setIsEmailLinkVerifiedModalOpen={order_store.setIsEmailLinkVerifiedModalOpen}
                     />
-                    <InvalidVerificationLinkModal
-                        invalid_verification_link_error_message={order_store.verification_link_error_message}
-                        is_invalid_verification_link_modal_open={order_store.is_invalid_verification_link_modal_open}
-                        setIsInvalidVerificationLinkModalOpen={order_store.setIsInvalidVerificationLinkModalOpen}
-                        onClickGetNewLinkButton={() => order_store.confirmOrderRequest(id)}
```

Comment:
Same as this, should we refactor out modal props that receive handlers? Since it might affect reusability if other pages has different ways to handle the onClick events

Reply:
Good point. But i think for now we can since this modal is specific to orders only.